src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -15,12 +15,8 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Diagnostics.Contracts;
-using System.Linq;
 using System.Runtime.CompilerServices;
-using System.Runtime.ExceptionServices;
-using System.Security;
 using System.Threading.Tasks.Dataflow.Internal;
-using System.Threading.Tasks.Dataflow.Internal.Threading;
```

Comment:
Looks like removing this using broke the Windows_NT Debug AllConfigurations Build that didn't triggered on this PR so that is why it wasn't caught before merging.

The error is: 
```
error CS0246: The type or namespace name 'TimerCallback' could not be found (are you missing a using directive or an assembly reference?) [D:\j\workspace\AllConfigurat---c1564d53\src\System.Threading.Tasks.Dataflow\src\System.Threading.Tasks.Dataflow.csproj]
```

There is multiple PRs affected by this so I will go ahead and fix it. The reason it is failing is because this project has netstandard1.0 and netstandard1.1 configurations and that type lives under this namespace. 

cc: @stephentoub